### PR TITLE
[openmmtools] 0.8.3

### PR DESCRIPTION
**Pull request by jchodera**

### CODE REVIEW

File: conda-build-all [hunk 1]
```diff
@@ -256,8 +256,7 @@ def upload_to_binstar(m, python, numpy, username, max_retries=5,
     cmd.extend(['upload', '--no-progress', '-u', username, filename])
     if force:
         cmd.extend(['--force'])
-    # Force upload if extra: force_upload: True is specfied
-    elif (m.get_section('extra') and ('force_upload' in m.get_section('extra')) and (m.get_section('extra')['force_upload'])):
+    elif m.get_section('extra') and ('force_upload' in m.get_section('extra')) and (m.get_section('extra')['force_upload']=='True'):
```
Comment (rmcgibbo) [inline conda-build-all]: are you sure that force_upload is of type `str` with value 'True`'? I thought it was a bool.

Comment (jchodera) [inline conda-build-all]: Much to my surprise, this expression evaluated to `True` despite the YAML file containing `force_upload: False`. I'll double check its type and fix in a separate PR.

Comment (rmcgibbo) [inline conda-build-all]: Yaml and python are slightly different in how your write booleans. It should be `false`, not `False` in the YAML file.


http://www.yaml.org/refcard.html: ```
Language Independent Scalar types:
    { ~, null }              : Null (no value).
    [ 1234, 0x4D2, 02333 ]   : [ Decimal int, Hexadecimal int, Octal int ]
    [ 1_230.15, 12.3015e+02 ]: [ Fixed float, Exponential float ]
    [ .inf, -.Inf, .NAN ]    : [ Infinity (float), Negative, Not a number ]
    { Y, true, Yes, ON  }    : Boolean true
    { n, FALSE, No, off }    : Boolean false
```

Comment (jchodera) [inline conda-build-all]: Wait, that reference card doesn't say `false` is an option. Only one of `{ n, FALSE, No, off }`. Am I missing something?

Comment (jchodera) [inline conda-build-all]: Or are both `false` and `FALSE` permitted, but not `False`?

Comment (jchodera) [inline conda-build-all]: [This specification](http://yaml.org/type/bool.html) suggests that `False` is permitted.

Comment (jchodera) [inline conda-build-all]: I'm not sure why, but empirically, it seems that both `false` and `False` become `str` type---at least locally, on `osx` Python 3.5.

Comment (rmcgibbo) [inline conda-build-all]: Perhaps it's something else that's causing the problem then.

**Pull request merged**